charts/prometheus/templates/pdb.yaml

```
@@ -1,4 +1,4 @@
-{{- if .Values.server.podDisruptionBudget.enabled }}
+{{- if .Values.server.podDisruptionBudget }}
```

This will allow us to keep the toggle whilst enabling other spec fields.

```suggestion
{{- if .Values.server.podDisruptionBudget.enabled }}
{{- $pdbSpec := omit .Values.server.podDisruptionBudget "enabled" }}
```